documentation/guide-jpa-query.asciidoc

```diff
@@ -81,9 +140,9 @@ Pageable pageable = PageRequest.of(page, size);
 Page<DishEntity> dishes = dishRepository.findAll(pageable);
 ----
 
-=== Paging with QueryDSL
+=== Paging with Querydsl
 
-Pagination is also supported for dynamic queries with QueryDSL:
+Pagination is also supported for dynamic queries with Querydsl:
```

Comment:
QueryDSL is the right name. So why did that change?

Reply:
https://github.com/devonfw/devon4j/pull/508#discussion_r769461604

Reply:
https://github.com/devonfw/devon4j/pull/508#discussion_r769461604